core/domain/suggestion_services.py

```diff
@@ -2904,3 +2908,333 @@ def enqueue_contributor_ranking_notification_email_task(
+        margin-right: auto;"> <div class="title"> <h3 style="
+        font-family: 'Capriola'; font-style: normal; font-weight: 400;
+        font-size: 37px; line-height: 46px; text-align: center;
+        color: #00645C;">CERTIFICATE OF APPRECIATION</h3> </div> <br><br> 
+        <div class="content"> <div class="textContent" style="width: 90%;
+        position: absolute; top: 50%; left: 50%;
+        transform: translate(-50%, -50%);"> <p style="
+        font-family: 'Capriola'; font-style: normal; font-weight: 400;
+        font-size: 30px; line-height: 38px; text-align: center;
+        color: #8F9899;">WE GRATEFULLY ACKNOWLEDGE</p> <p style=" 
+        font-family: 'Capriola'; font-style: normal; font-weight: 400; 
+        font-size: 40px; line-height: 50px; text-align: center; 
+        color: #00645C; margin: 0; ">""" + username + """</p> <p style=" 
+        font-family: 'Capriola'; font-style: normal; font-weight: 400; 
+        font-size: 26px; line-height: 32px; text-align: center; 
+        color: #8F9899;"> for your dedication and time in translating 
+        <span style="color: #00645C;">Oppia’s</span> basic maths lessons 
+        to <span style="color: #00645C;">""" + language_description + """
+        </span> which will help our """ + language_description + """
+        - speaking learners better understand the lessons.
+        <br><br> 
+        This certificate confirms that """ + username + """ has contributed 
+        """ + str(hours_contributed) + """ hours’ worth of translations
+        from """ + from_date.strftime('%d %b %Y') + """ to 
+        """ + to_date.strftime('%d %b %Y') + """ </p> </div> 
+        <div style="display:block; position: absolute; top: 80%; 
+        left: 50%; transform: translate(-50%, -50%); width: 100%;">
+        <div class="signature" style="width: 50%; float:left; 
+        text-align: center;"><div style="
+        font-family: Brush Script MT, Brush Script Std, cursiv;
+        font-size: 36px; line-height: 10px; margin-top: 0px;
+        color: #000000;">"""+ signature +"""</div> <center> 
+        <p class="line" style="width: 186px; height: 0px; border: 1px 
+        solid #00645C;"></p> </center> <p style="font-family: 'Roboto'; 
+        font-style: normal; font-weight: 400; font-size: 24px; 
+        line-height: 28px; margin-top: 0px; color: #8F9899;">SIGNATURE</p> 
+        </div> <div class="date" style="width: 50%; float:left; 
+        text-align: center;"><div style="font-family: 'Roboto';
+        font-size: 24px; color: #000000; line-height: 10px;">
+        """ + date + """</div><center> <p class="line" style="
+        width: 186px; height: 0px; border: 1px solid #00645C;"></p> 
+        </center> <p style="font-family: 'Roboto'; font-style: normal; 
+        font-weight: 400; font-size: 24px; line-height: 28px; 
+        margin-top: 0px; color: #8F9899;">DATE</p> </div> </div> </div>
+        </div> </div> </div> </body> </html>
+    """
```

Comment:
Can the formatting of this be improved for a better readability? Now this is just a blob of text where we also have some custom parts.